react/src/loginForm.js

```diff
@@ -0,0 +1,76 @@
+import { Form, Input, Button, Checkbox } from 'antd';
+import React from 'react';
+
+class LoginForm extends React.Component {
+  constructor(props) {
```

Comment:
I was expecting that you setState here. But I don't see a set state for this component.